src/app/app.routes.ts

```diff
@@ -15,8 +16,19 @@ export const routes: Routes = [
     canMatch: [authGuard],
   },
   { path: 'search', component: SearchPageComponent, pathMatch: 'full' },
-  { path: 'signin', component: SigninPageComponent, canMatch: [authGuard], pathMatch: 'full' },
-  { path: 'signup', component: SignupPageComponent, canMatch: [authGuard], pathMatch: 'full' },
+  { path: 'trip/:id', component: TripPageComponent },
```

Comment:
```suggestion
  { path: 'trip/:id', component: TripPageComponent, pathMatch: 'full' },
```
Добавь пасматч, чтобы левые роуты туда не попадали.